package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "d3",
-  "version": "5.16.0",
+  "version": "6.0.0-alpha.0000",
```

Comment:
Why not -rc.1?

Reply:
it was the end of a long session, I just wanted to check it worked ;-)